cli/src/main/java/com/liferay/blade/cli/BladeSettings.java

```diff
@@ -35,6 +38,28 @@ public BladeSettings(File settingsFile) throws IOException {
 		if (_settingsFile.exists()) {
 			load();
 		}
+		else {
+			if (WorkspaceUtil.isWorkspace(settingsFile)) {
+				File workspaceDirectory = WorkspaceUtil.getWorkspaceDir(settingsFile);
+
+				File pomFile = new File(workspaceDirectory, "pom.xml");
+
+				if (pomFile.exists()) {
+					String canonicalPath = settingsFile.getCanonicalPath();
+
+					String questionString =
+						"Blade has detected that " + canonicalPath + " does not exist. Should blade create this file?" +
```

Comment:
Lets use this phrase... `WARNING: blade commands will not function properly in a Maven workspace unless the blade profile is set to "maven".  Should the settings for this workspace be updated?`